submissions/adeyemiJuwon/dom/newsletter.js

```diff
@@ -0,0 +1,35 @@
+let submitForm = document.getElementById('newsletter');
+let email = document.getElementById('email');
+
+
+
+function required(field, event){
```

Comment:
You can as well do all this in one function rather than 3 different functions since it is just the email field you are validating